matterbridge.toml.template

```diff
@@ -206,3 +213,7 @@
 	[[gateway.inout]]
 	account="slack.parti"
 	channel="fact-checking-project"
+
```

Comment:
This will add a new gateway to the existing bridge for disinformation.  Instead, what you want is to create a new bridge between the edu-open-stu-association channels in g0v-tw Slack and student DAO discord.

You need to add the "edu-open-stu-association" gateway that was previously removed here https://github.com/g0v/bridge/pull/18 to do that:

```
[[gateway]]
	name="edu-open-stu-association"
	enable=true

	[[gateway.inout]]
	account="slack.g0v-tw"
	channel="edu-open-stu-association"

	[[gateway.inout]]
	account="discord.studentdao"
	channel="edu-open-stu-association"
```